docs/Example-Usage.md

```diff
@@ -0,0 +1,41 @@
+# Example Usage
+
+This is an example of the most common use case of this SDK - fetching operations and approving them.
+
+## SDK Integration
+
+Follow the [SDK Integration](./SDK-Integration.md) tutorial for SDK installation.
+
+## Example Code
+
+```swift
+// PowerAuth instance needs to be configured and a user-activated instance.
+// More about PowerAuth SDK can be found here: https://github.com/wultra/powerauth-mobile-sdk
+
+func exampleUsage(powerAuth: PowerAuthSDK) {
+do {
+    let mtoken = try powerauth.createWultraMobileToken(acceptLanguage: "de") // create the WultraMobileToken instance and set "requested content" to german language (default is english - "en")
+
+    let operations = mtoken.operations.getOperations { result in   
+        
+        switch result {
+        case .success(let ops):
+            let auth = PowerAuthAuthentication.possessionWithPassword(password: "1234") // simulate that user entered PIN 1234
+            mtoken.operations.authorize(operation: ops.first!, with: auth) { result in
```

Comment:
add comment that we expect at least 1 operation in the list for example purposes